src/Metrics.php

```diff
@@ -138,7 +145,7 @@ public function track(Request $request, $response): void
     public function constructPayload(Request $request, $response): array
     {
         $request_start = (!defined('LARAVEL_START')) ? LARAVEL_START : $_SERVER['REQUEST_TIME_FLOAT'];
-        $group = ($this->group)($request);
+        $group = $this->group_handler::constructGroup($request);
 
         if (!array_key_exists('id', $group)) {
```

Comment:
I feel these two checks are redundant, maybe the slight difference in errors is noticeable to the consumer.

Reply:
I'd rather keep them separate so we can show contextual errors for those two different misconfiguration situations.